packages/native/types/base.d.ts

```diff
@@ -3,25 +3,21 @@
 
 import { ComponentPropsWithoutRef, ComponentType } from 'react'
 import * as RN from 'react-native'
-import { PropsOf } from '@emotion/core'
```

Comment:
the `ComponentProps` construct in the official React typings should be the standard for props extraction on new typing efforts, it differs slightly from the `PropsOf` in core but in ways that produce more accurate props (specifically for `react-native`). In the case of `react-native`, i can't think of a reason the `ref` prop would be used in styles so i went with the `WithoutRef` variant, though i'm totally willing to stand down on that assumption if there is any reason whatsoever to use `ref` in styles (my understanding is that it's only used for native prop manipulation).

Reply:
an added note, we may still need/want to `import '@emotion/core'` here so that the side effects of core are automatically applied when importing `native`.